gf2.cpp

```diff
@@ -1003,6 +1016,22 @@ void CommandDeleteBreakpoint(void *_index) {
 	DebuggerSend(buffer, true, false);
 }
 
+void CommandDisableBreakpoint(void *_index) {
```

Comment:
We could merge CommandDisableBreakpoint() and CommandEnableBreakpoint() into a single function CommandToggleDisableBreakpoint(), but i don't know if keep them separate could comme in handy in the futur?